appserver/jdbc/jdbc-runtime/src/test/java/org/glassfish/jdbcruntime/deployment/annotation/handlers/DataSourceDefinitionImpl.java

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
This file is not necessary. Please use Mockito to mock / stub this out

Reply:
I realize this is a refactor, but it's opportunity to delete code not introduce more files

Reply:
Introduced mockito. Removed unnecessary files. Integrated test into existing one.